dist/vue-toastr.umd.js

```diff
@@ -380,7 +380,7 @@
   const __vue_script__$1 = script$1;
 
   /* template */
-  var __vue_render__$1 = function () {var _vm=this;var _h=_vm.$createElement;var _c=_vm._self._c||_h;return _c('div',{class:_vm.classNames,staticStyle:{"display":"block"},style:(_vm.data.style),on:{"click":function($event){return _vm.clicked()},"mouseover":_vm.onMouseOver,"mouseout":_vm.onMouseOut}},[(_vm.progressbar)?_c('toast-progress',{ref:"progressBar",attrs:{"percent":_vm.progressBarPercent}}):_vm._e(),_vm._v(" "),_c('div',{staticClass:"toast-title",domProps:{"innerHTML":_vm._s(_vm.data.title)}}),_vm._v(" "),_c('div',{staticClass:"toast-message",domProps:{"innerHTML":_vm._s(_vm.data.msg)}})],1)};
+  var __vue_render__$1 = function () {var _vm=this;var _h=_vm.$createElement;var _c=_vm._self._c||_h;return _c('div',{class:_vm.classNames,staticStyle:{"display":"block"},style:(_vm.data.style),on:{"click":function($event){return _vm.clicked()},"mouseover":_vm.onMouseOver,"mouseout":_vm.onMouseOut}},[(_vm.progressbar)?_c('toast-progress',{ref:"progressBar",attrs:{"percent":_vm.progressBarPercent}}):_vm._e(),_vm._v(" "),(_vm.data.title)?_c('div',{staticClass:"toast-title",domProps:{"innerHTML":_vm._s(_vm.data.title)}}):_vm._e(),_vm._v(" "),(_vm.data.msg && !_vm.$slots.default)?_c('div',{staticClass:"toast-message",domProps:{"innerHTML":_vm._s(_vm.data.msg)}}):_vm._e(),_vm._v(" "),(_vm.$slots.default)?_c('div',{staticClass:"toast-message"},[_vm._t("default")],2):_vm._e()],1)};
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Split this 785 characters long line (which is greater than 180 authorized). ([javascript:LineLength](https://sonar.igentify.net/coding_rules#rule_key=javascript%3ALineLength))

[See it in SonarQube](https://sonar.igentify.net/project/issues?id=vuet&pullRequest=2&issues=AWz13RYQAaEKZWHEUhIH&open=AWz13RYQAaEKZWHEUhIH)

[ISSUE_KEY]: AWz13RYQAaEKZWHEUhIH